### PR TITLE
Feature: events

### PR DESCRIPTION
**Pull request by memelopez**
## This PR implements: 🆕 👨‍💻 🔥 👍 

- No [linter errors](https://github.com/microverseinc/linters-config). ✔️ 
- Correct use of [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). ✔️ 
- Work is documented in a [professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). ✔️ 
- Follows [JS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). ✔️ 
- **Created a new directory** called `logic` inside project. ✅ 
-  **Copied the files** `calculate.js` and `operate.js` into `logic/`. ✅ 
- **Installed** [big.js](https://www.npmjs.com/package/big.js) using NPM. ✅ 
- **Analized the files** `calculate.js` and `operate.js` to understand them. ✅ 
- **Imported files** to `Calculator.js` component. ✅ 
- **Implemented click event handlers** using the math logic.  ✅ 
- **Tested app in browser.** Calculator performs math operations. ✅

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @memelopez, Kindly remove the broken readme logo and update the logo for your math magician project, and add content for all sections of your readme,  and you are currently missing the key and future features section in your readme

**Comment by prolajumokeoni** [comment]
Hi @memelopez 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that  does not have a descriptive and professional readme 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.js [hunk 1]
```diff
@@ -20,36 +21,129 @@ Output.defaultProps = {
...
+      next,
+      operation,
+    };
+    const newCalcObj = calculate(calcObj, value);
+    this.setState({
+      total: newCalcObj.total,
+      next: newCalcObj.next,
+      operation: newCalcObj.operation,
+    });
+  }
```
Comment (AmaduKamara) [inline src/components/Calculator.js]: - [x] For this project, you're required to make use of the `useSatet` hook to manage the calculator app state. Kindly implement the use of useState hook. 👍🏽

HINT:
```javascript
// ./src/components/Calculator.js

import React, { useState } from 'react';
import calculate from ../logic; // Import math logic

function Calculator() {
    const [total, setTotal] = useState(0);
    return {
      <div>
       // Calculator app 
      </div>
    };
}
```

As required here:
Screenshot 2023-02-25 at 3 06 46 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @memelopez,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍🏽 
✅ Linter checks passed 👍🏽
✅ Used calculator logic files provided 👍🏽 
✅ Good descriptive PR 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly use the React `useState` hook for the calculator app state as required for this project 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Olamarx — APPROVED** [review]
Hi @memelopez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- useState hook ✔️
- Linters passed ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**